src/OneExplorer/OneExplorer.ts

```diff
@@ -1026,8 +1025,65 @@ export class OneTreeDataProvider implements vscode.TreeDataProvider<Node> {
   }
 
   /**
-   * Create ONE configuration file for a base model
-   * Input box is prefilled as <base model's name>.cfg
+   * Select the option for the configuration you want to create
+   * Return information about the selected option
+   * Default option is information about .cfg
+   *
+   * @param modelName A base model's name
+   * @param extName A base model's extension name
+   *
+   */
+  private async generateCfgInfo(
+    modelName: string,
+    extName: string
+  ): Promise<CfgInfo> {
+    const options = [
+      { label: ".cfg", description: "configuration file of onecc compiler" },
+    ];
+
+    if (extName === "tflite") {
+      options.push({
+        label: ".edgetpucfg",
+        description: "configuration file of edgetpu compiler",
+      });
+    }
+
+    const selectedOption = await vscode.window.showQuickPick(options);
+    const selectedLabel = selectedOption ? selectedOption.label : ".cfg";
```

Comment:
When there is no selectedOption, it should just return and exit.

Because of this line, it behaves unexpectedly: When the quickpick asks me whether to create ".cfg" or "edgetpucfg" and I hit `esc` to exit, but it still creates .cfg.